packages/terra-table/tests/wdio/table-spec.js

```diff
@@ -0,0 +1,93 @@
+Terra.describeViewports('Table', ['medium', 'large'], () => {
+  describe('Default Table', () => {
+    it('Validates the default table', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/table/default-table');
+      Terra.validates.element('default-table', '#default-terra-table');
+    });
+  });
+
+  describe('No Interaction Table', () => {
+    it('Validates the default table is not interactable', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/table/no-interaction-table');
+      browser.keys(['Tab']);
+      $('#focused-button').isFocused();
```

Comment:
You should be doing an expect on this, right?  This code does not actually validate that this button is focused.

Reply:
Fixed.